configtx/msp.go

```diff
@@ -74,6 +75,393 @@ type MSP struct {
 // YEAR is a time duration for a standard 365 day year.
 const YEAR = 365 * 24 * time.Hour
 
+// OrganizationMSP encapsulates the configuration functions used to modify an organization MSP.
+type OrganizationMSP struct {
+	configGroup *cb.ConfigGroup
+}
+
+// CreateMSPCRL creates a CRL that revokes the provided certificates
+// for the specified organization's msp signed by the provided SigningIdentity.
+func (m *MSP) CreateMSPCRL(signingIdentity *SigningIdentity, certs ...*x509.Certificate) (*pkix.CertificateList, error) {
+	return m.newMSPCRL(signingIdentity, certs...)
+}
+
+// Configuration returns the MSP value for an orderer organization in the
```

Comment:
`Configuration returns the MSP value for an organization in the updated config.`

Reply:
🛠️